### PR TITLE
Update James SHA1

### PR DESCRIPTION
**Pull request by quantranhong1999**
for TMail 0.6.3 release

### CODE REVIEW

File: upgrade-instructions.md [hunk 1]
```diff
@@ -14,4 +14,70 @@ Note: this section is in progress. It will be updated during all the development
...
\ No newline at end of file
+(No changes yet.)
+
+## 0.6.3 version
+
+Change list:
+
+- [Improve indexing of email address domains](#improve-indexing-of-email-address-domains)
+
+### Improve indexing of email address domains
```
Comment (Arsnael) [inline upgrade-instructions.md:25]: I think that upgrade instruction should be part of apache james and not necessary here..?

Comment (quantranhong1999) [reply upgrade-instructions.md:25]: We will need documentation upgrade instructions for `JAMES-3827 Backport to ElasticSearch 6.x` anyway.

Comment (Arsnael) [reply upgrade-instructions.md:25]: Usually putting them on the changelog but... it's true that we don't use this upgrade-instructions in this project until now. Deserves a discussion :)

Comment (chibenwa) [reply upgrade-instructions.md:25]: No not in james as we migrated from OpenSearch.

In TMail it is different as we have an intermediate OpenSearch release, which thus need an upgrade instruction.

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Pull request merged**